README.md

```diff
@@ -1 +1,32 @@
-## Battleship with Java
\ No newline at end of file
+## Battleship with Java
+
+ Battleship game built with  Java 11 and  Spring Boot 2.5.6
+
+## Installation
+1.Clone the repository to your computer.
+
+2.Import the app into your IDE through build.gradle file.
+
+## Start
+To start the application run the BattleshipApplication(./src/main/java/com.game.Battleship/BattleshipApplication)
+
+or run in te terminal:
+
+``` ./gradlew bootRun```
+
+The server is initialized with port: 8080(http)
+
+For the home page go to:
+
+http://localhost:8080/home 
+
+
+## Run tests
+
+To start tests, run BattleshipApplicationTests(./src/test/java/com.game.Battleship/BattleshipApplicationTests)
+
+or run in the terminal:
+
+```./gradlew test```
+
+
```

Comment:
Two blank lines at the end not necessary. Try to keep this consistent with all files. On my project, for instance, our linter forces a single blank line at the end of each file.